.github/workflows/pull-request-labeler.yml

```diff
@@ -0,0 +1,23 @@
+name: "Label PRs"
+on:
+- pull_request_target
+
+jobs:
+  apply_labels:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: release-drafter/release-drafter@v5
+      with:
+        name: v${{ github.event.inputs.tag }}
+        tag: pr-label-release-to-delete
```

Comment:
nit: can we rename this to something self explanatory like `temp-tag-created-by-pull-request-labeler` or `pull-request-labeler-tag`

Reply:
Updated to `temp-tag-created-by-pr-auto-labeler` for clarity!